doc/src/guides/solving/solve-equation-algebraically.md

```diff
@@ -112,10 +108,9 @@ $expression = 0$.
 
 ### Put your equation into `Eq` form
 
-Put your equation into `Eq` form, then solve the `Eq`.
-This approach is convenient if you are interactively solving an equation
-which you already have in the form of an equation,
-or which you think of as an equality.
+Put your equation into `Eq` form, then solve the `Eq`. This approach is
+convenient if you are interactively solving an equation which you already have
+in the form of an equation, or which you think of as an equality.
```

Comment:
```suggestion
in the form of an equation, or which you think of as an equality. It also
help to prevent sign errors when subtracting one side from the other.
```